packages/plugin-commons/web/src/Base/createBaseComponent.jsx

```diff
@@ -274,6 +270,10 @@ const createBaseComponent = ({
       const { width: currentWidth, height: currentHeight } = componentData.config || {};
       const { width: initialWidth, height: initialHeight } = settings || {};
       const isEditorFocused = selection.getHasFocus();
+      const isOneBlockSelected = selection.getStartKey() === selection.getEndKey();
+      const isSelected = blockProps.isFocused && !isOneBlockSelected;
```

Comment:
When one block is selected, it's still selected? 
```suggestion
      const isSelected = blockProps.isFocused
```

Reply:
I see 2 use cases here:
1. the block is part of selection (multiple blocks selected), this case is only to support border on every plugin.
2. the block is focused (only the block is selected).

Reply:
And no, if there is one block selected and the block is focused - than the block is focused.